parsl/executors/workqueue/executor.py

```diff
@@ -369,36 +382,61 @@ def WorkQueueCollectorThread(collector_queue=multiprocessing.Queue(),
 
 
 class WorkQueueExecutor(ParslExecutor):
-    """Executor to use Workqueue batch system
+    """Executor to use Work Queue batch system
+
+    The WorkQueueExecutor system utilizes the Work Queue framework to 
+    efficiently delegate Parsl apps to remote machines in clusters and 
+    grids using a fault-tolerant system. Users can run the
+    work_queue_worker program on remote machines to connect to the
+    WorkQueueExecutor, and Parsl apps will then be sent out to these 
+    machines for execution and retrieval.
+
+
+        Parameters
+        ----------
 
         label: str
-            a human readable label for the executor, unique
-            with respect to other executors.
+            A human readable label for the executor, unique
+            with respect to other Work Queue master programs
+
         working_dir: str
-            the directory location for parsl to run the process
+            Location for Parsl to perform app delegation to the Work
+            Queue system
+
         managed: bool
+            If this executor is managed by the DFK or externally handled
+
         project_name: str
-            workqueue process name
+            Work Queue process name
+
         project_password: str
-            password for the work queue project
+            Optional password for the Work Queue project
+
         project_password_file: str
-            password file for the work queue project
+            Optional password file for the work queue project
+
         port: int
-            port to connect to
+            TCP port for Work Queue workers to connect to 
```

Comment:
Where is this port if I'm running a parsl submission program on machine A and some workers on machines B and C. I think it is on machine A?

Reply:
Yes, on A